### ISSUE TITLE
REQUEST_COMMAND documentation is out of date

### ISSUE THREAD

**Issue by philip-wernersbach** [orientechnologies/orientdb#3508]
The documentation for REQUEST_COMMAND at https://github.com/orientechnologies/orientdb/wiki/Network-Binary-Protocol#request_command appears to be slightly out of date. I'll update this with more specifics in the morning.

**Comment by philip-wernersbach** [comment]
Right now the Network Binary Protocol page lists the REQUEST_COMMAND request as having this format:
```
Request: (mode:byte)(class-name:string)(command-payload-length:int)(command-payload)
```
This does not work with OrientDB 2.0. When I tried implementing synchronous idempotent commands in an OrientDB driver I am writing, the server rejected this format. So I ran the same command in the OrientDB console and took a look at the Wireshark output for the session and it appears that the format for REQUEST_COMMAND is now this:
```
Request: (mode:byte)(command-payload-length:int)(class-name:string)(command-payload)
```
Where `command-payload-length` is the length of the `class-name` field plus the `command-payload` field.

It also appears that the [SQL (Select) Query](https://github.com/orientechnologies/orientdb/wiki/Network-Binary-Protocol-Commands) format now omits the `serialized-params` field, or at least the OrientDB console does not send them.

I gathered this information by reverse engineering through Wireshark dumps, so I could be totally wrong here.
